apps/storybook/src/overlays/Spotlight.stories.tsx

```diff
@@ -214,7 +214,13 @@ export const Basic: Story<{ items?: Item[] } & Item> = {
                         </Box>
                       </SpotlightEmpty>
 
-                      <SpotlightItem item={item}>
+                      <SpotlightItem
```

Comment:
We can actually replace this with SpotlightRadioItem - missed replacing this in last PR

Reply:
I also thought of using it vai, but wasn't sure if it was okay to use RadioItem.